src/site/xdoc/downloads.xml

```diff
@@ -70,26 +70,26 @@ under the License.
     </tr>
     <tr>
       <td style="test-align: left">
-        2.5.2
+        2.5.3
       </td>
       <td style="test-align: left">
-        2022/12/03
+        2023/02/05
```

Comment:
Thank @Apache9 , I'm still using 2023/02/05 because my time zone is PST and have 4 hours remaining before end of today, let's see how it goes